authorization/index.md

```diff
@@ -187,18 +188,32 @@ cross-site request forgery or session fixation attacks.
 
       </td>
     </tr>
+     <tr>
+      <td><code>aud</code></td>
+      <td><span class="label label-default">recommended</span></td>
+      <td>
+
+URL of the EHR resource server from which the app wishes to retrieve a FHIR resource.  This parameter SHOULD be used to prevent presenting a bearer token to a counterfeit resource server (which then could use it to retrieve the authorized resource from the authentic resource server).  
+
+      </td>
+    </tr>
   </tbody>
 </table>
 
+The app MUST use an unpredictable value for the state parameter with at least 128 bits of entropy. The app MUST validate the value of the state parameter upon return to the redirect URL and MUST ensure that the state value is securely tied to the user’s current session (e.g., by relating the state value to a session identifier issued by the app).
+The app SHOULD limit the grants, scope, and period of time requested to the minimum necessary.   The app MAY digitally sign the request, with the EHR validating the signature.  
```

Comment:
Can you remove the app-signing piece from this particular PR? I think that will need separate discussion, and I'd rather note mix it into the other five changes that are already being made here.

(Specifically: if we are going to describe this we'll have to explain how it works. This PR currently just says "The app MAY digitally sign the request" -- with no details about the signature process. For now let's just remove this sentence and then deal with it in a separate discussion.)